apps/frontend-react/src/hooks/useQueryTransmission.ts

```diff
@@ -0,0 +1,34 @@
+import { DbObject, ITransmission } from "@emptystream/shared";
+import { useState } from "react";
+import TransmissionQueries from "../queries/TransmissionQueries";
+
+export interface UseQueryTransmissionProps {
+  id: string;
+  initialData?: DbObject<ITransmission>;
+}
+
+export default function useQueryTransmission({ id, initialData }: UseQueryTransmissionProps) {
```

Comment:
Wait. Why do we need this anymore? Why isn't this just implemented directly into useQuerySingle?

Reply:
I think this is an artifact from when TransmissionQueries was going to just return queryOptions, rather than the queries themselves. We can prob implement this into TransmissionQueries now.